### PR TITLE
Add general info about Bower

### PR DESCRIPTION
**Pull request by warpech**
@PatrykSzwer this is mostly to explain what I am doing in:

- https://github.com/StarcounterApps/Products/pull/53
- https://github.com/StarcounterApps/KitchenSink/pull/151

in order to solve custom elements that need updating because of https://github.com/Starcounter/RebelsLounge/issues/72

any questions?

@tomalec any comments?

### CODE REVIEW

File: bower.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Bower offers several ways to install packages:
+
+```
+# Using the dependencies listed in the current directory's bower.json
+bower install
+# Using a local or remote package
+bower install <package>
+# Using a specific version of a package
+bower install <package>#<version>
```
Comment (joozek78) [inline bower.md]: I would add an example of installing a juicy component from github (from a specific branch). I think it's at the same time useful and not-so-obvious

**Comment by warpech** [comment]
@joozek78 done

**Comment by tomalec** [comment]
LGTM

**Comment by warpech** [comment]
Thanks for the review!

**Pull request merged**